apps/hash-frontend/src/pages/inbox.page.tsx

```diff
@@ -0,0 +1,225 @@
+import { simplifyProperties } from "@local/hash-isomorphic-utils/simplify-properties";
+import {
+  extractEntityUuidFromEntityId,
+  extractOwnedByIdFromEntityId,
+} from "@local/hash-subgraph";
+import {
+  Container,
+  Skeleton,
+  styled,
+  Table as MuiTable,
+  TableBody,
+  TableCell as MuiTableCell,
+  tableCellClasses,
+  TableHead,
+  TableRow as MuiTableRow,
+  tableRowClasses,
+  Typography,
+} from "@mui/material";
+import { FunctionComponent, useMemo } from "react";
+
+import { useUserOrOrgShortnameByOwnedById } from "../components/hooks/use-user-or-org-shortname-by-owned-by-id";
+import { constructPageRelativeUrl } from "../lib/routes";
+import { getLayoutWithSidebar, NextPageWithLayout } from "../shared/layout";
+import {
+  Notification,
+  useNotifications,
+} from "../shared/notifications-context";
+import { Button, Link } from "../shared/ui";
+
+const Table = styled(MuiTable)(({ theme }) => ({
+  borderCollapse: "separate",
+  borderSpacing: 0,
+  background: theme.palette.common.white,
+  borderRadius: "8px",
+  borderColor: theme.palette.gray[30],
+  borderStyle: "solid",
+  borderWidth: 1,
+  overflow: "hidden",
+}));
+
+const TableRow = styled(MuiTableRow)(() => ({
+  [`&:last-child .${tableCellClasses.body}`]: {
+    borderBottom: "none",
+  },
+  [`&:first-child .${tableCellClasses.head}`]: {
+    "&:first-child": {
+      borderTopLeftRadius: "8px",
+    },
+    "&:last-child": {
+      borderTopRightRadius: "8px",
+    },
+  },
+  [`&:last-child .${tableCellClasses.body}`]: {
+    "&:first-child": {
```

Comment:
> client.js:1 The pseudo class ":first-child" is potentially unsafe when doing server-side rendering. Try changing it to ":first-of-type".

(probably will cause playwright test failure due to unexpected console logs)

Reply:
Ah thanks for flagging, have addressed in https://github.com/hashintel/hash/pull/3511/commits/9d765d58258069a841d4c65499defede3a12dafb